src/Scenes/OptionsScene.js

```diff
@@ -36,27 +33,28 @@ export default class OptionsScene extends Phaser.Scene {
 
     this.updateAudio();
 
-    this.menuButton = new Button(this, 400, 500, 'menuButton', 'menuButtonPressed', 'Title');
+    this.menuButton = new Button(this, 550, 500, 'menuButton', 'menuButtonPressed', 'Title');
     this.updateAudio();
   }
 
   updateAudio() {
     if (this.model.musicOn === false) {
-      this.musicButton.setTexture('box');
+      this.musicButton.setAlpha(0.001);
```

Comment:
This was my idea, invisible objects (or even ones with 0 alpha) stop being interactive for reasons I dont have time to divine.